redbot/core/commands/help.py

```diff
@@ -289,13 +290,32 @@ def group_embed_fields(fields: List[EmbedField], max_chars=1000):
         pages = []
 
         page_char_limit = await ctx.bot._config.help.page_char_limit()
+        page_char_limit = min(page_char_limit, 5990)  # Just in case someone was manually...
+
+        author_info = {"name": f"{ctx.me.display_name} Help Menu", "icon_url": ctx.me.avatar_url}
+
+        # Offset calculation here is for total embed size limit
+        # 20 accounts for# *Page {i} of {page_count}*
```

Comment:
Should half of this be on a separate line or is there just a random extra `#`?

Reply:
Was intended to split the literal code being accounted for from the rest of the comment.